source/_posts/2023-06-07-release-20236.markdown

```diff
@@ -0,0 +1,867 @@
+measurement, there is currently no way to get text sensors from IHC working.
+
+This is a breaking fix because if you currently have temperature sensors
+without a unit of measurement set, you will have to set this for the sensors.
+
+([@dingusdk] - [#91096]) ([documentation](/integrations/ihc))
+
+[@dingusdk]: https://github.com/dingusdk
+[#93054]: https://github.com/home-assistant/core/pull/93054
+
+{% enddetails %}
+
+{% details "LG Netcast" %}
+
+LG Netcast media players no longer have fake (assumed) `playing` or `paused`
+states, and those are replaced with the `on` state.
+
+If you used the `playing` or `paused` state in your automations, you will need
+to update them to match this change.
+
+([@Drafteed] - [#93543]) ([documentation](/integrations/lg_netcast))
+
+[@Drafteed]: https://github.com/Drafteed
+[#93543]: https://github.com/home-assistant/core/pull/93543
+
+{% enddetails %}
+
+{% details "Meteorologisk institutt (Met.no)" %}
+
+The weather forecast provided by the weather entity has changed. The current
+day has been added to the forecast. If your automation or scripts relied
+on the first day to be tomorrow, you would need to adjust.
+
+([@mmalina] - [#91891]) ([documentation](/integrations/met))
+
+[@mmalina]: https://github.com/mmalina
+[#91891]: https://github.com/home-assistant/core/pull/91891
+
+{% enddetails %}
+
+{% details "MQTT" %}
+
+MQTT sensor configurations that still use `last_reset_topic` will fail. The
+support was deprecated in Home Assistant 2021.11 and is now removed. Users
+should use ` last_reset_value_template` instead.
+
+([@jbouwh] - [#93792]) ([documentation](/integrations/mqtt))
+
+[@jbouwh]: https://github.com/jbouwh
+[#93792]: https://github.com/home-assistant/core/pull/93792
+
+{% enddetails %}
+
+{% details "Nextcloud" %}
+
+The previously deprecated YAML configuration of the Nextcloud
+integration has been removed.
+
+Nextcloud is now configured via the UI. Any existing YAML
+configuration has been imported in previous releases and can now be safely
+removed from your YAML configuration files.
+
+([@gjohansson-ST] - [#93550]) ([documentation](/integrations/nextcloud))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#93550]: https://github.com/home-assistant/core/pull/93550
+
+{% enddetails %}
+
+{% details "Obihai" %}
+
+The previously deprecated YAML configuration of the Obihai
+integration has been removed.
+
+Obihai is now configured via the UI. Any existing YAML
+configuration has been imported in previous releases and can now be safely
+removed from your YAML configuration files.
+
+([@gjohansson-ST] - [#93549]) ([documentation](/integrations/obihai))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#93549]: https://github.com/home-assistant/core/pull/93549
+
+{% enddetails %}
+
+{% details "Persistent notifications" %}
+
+Persistent notifications are no longer stored in the state machine. This means
+that we no longer create an entity for each persistent notification.
```

Comment:
This doesn't mention what users need to do. Have we decided if we're going to add a template helper?

Reply:
@MartinHjelmare They can do nothing. There is nothing; the template helper is in draft, not this release.